anydex/test/wallet/stellar/test_stellar_wallet.py

```diff
@@ -0,0 +1,308 @@
+import abc
+import time
+from datetime import datetime
+
+from ipv8.util import succeed
+from sqlalchemy.orm import session as db_session
+from stellar_sdk import Keypair
+
+from anydex.wallet.stellar.xlm_db import Secret, Transaction
+from anydex.wallet.stellar.xlm_wallet import StellarWallet, StellarTestnetWallet
+from anydex.wallet.wallet import InsufficientFunds
+from anydex.test.base import AbstractServer
+from anydex.test.util import MockObject, timeout
+
+
+class TestAbstractStellarWallet(metaclass=abc.ABCMeta):
```

Comment:
Nice implementation of testing btw. I get that the abstract tests are implemented in the concrete test classes. I want to clarify though, at which point are the non-abstract test methods ran?

Reply:
I'm not sure you mean by at which point they are ran. They are ran when the `TestStellarTestnetWallet` or `TestStellarWallet` test classes are run.